src/controllers/bookings.cr

```diff
@@ -39,6 +39,7 @@ class Bookings < Application
 
     query = query
       .order_by(:booking_start, :desc)
+      .where(deleted: false)
```

Comment:
Please allow for a param `?deleted=true` that defaults to false so we can return deleted bookings via the API